kube/resources/worker.yaml

```diff
@@ -184,6 +184,32 @@ spec:
                 configMapKeyRef:
                   name: airbyte-env
                   key: JOB_KUBE_MAIN_CONTAINER_IMAGE_PULL_POLICY
+            # todo: add other state storage keys
+            - name: STATE_STORAGE_MINIO_BUCKET_NAME
```

Comment:
we can add in a follow up PR - should also update the https://docs.airbyte.com/operator-guides/configuring-airbyte#kubernetes-only with the state env vars.